sdk/include/opentelemetry/sdk/logs/logger_provider.h

```diff
@@ -107,14 +109,20 @@ class LoggerProvider final : public opentelemetry::logs::LoggerProvider
    */
   const opentelemetry::sdk::resource::Resource &GetResource() const noexcept;
 
-private:
-  // A pointer to the processor stored by this logger provider
-  std::shared_ptr<sdk::logs::LoggerContext> context_;
+  /**
+   * Shutdown the log processor associated with this log provider.
+   */
+  bool Shutdown() noexcept;
```

Comment:
shouldn't we have timeout here?

Reply:
The Shutdown of tracer privoder do not have timeour.I think they should have the same arguments.